DTS-STN/canadian-dental-care-plan
# Renaming `stub-sin-editor` to `stub-login` and refactoring stub login to include fields for `sid` and `sub`

### Description
Separating fields for `sid` and `sub` claims increases flexibility in case developers wish to specify different values for these claims.

Also included some very basic field validation and default values for `sid` and `sub` fields. Next-gen field validation for SIN, `sid` and `sub` are not included because this page is for development purposes.

### Screenshots (if applicable)
**Before:**
![image](https://github.com/user-attachments/assets/8f93d795-26e7-4519-83dd-0184ce82a023)

**After:**
![image](https://github.com/user-attachments/assets/ee15cee5-7135-4e28-8cbf-9d5054c69f5b)

### Checklist

### Test Instructions
Run application and navigate to `/en/stub-login`. Use the following values and hit "Login"
- SIN: 800011819
- `sid`: 00000000-0000-0000-0000-000000000000
- `sub`: 76c48130-e1d4-4c2f-8dd0-1c17f9bbb4f6 (Note that currently this value must correspond to a `userAttribute` found in `mocks/db.ts`)

### Additional Notes
There's a TODO in `stub-login` that will be removed in a future PR.
